### DISCUSSION TITLE
Metrics by path

### DISCUSSION THREAD

**Discussion by skarasev** [envoyproxy/gateway#4194] [Q&A]
I d like to have request path and method in my proxy metrics. At least for a manually configured list of paths.
For now I am thinking the way to do it is an EnvoyPatchPolicy with _envoy.bootstrap.wasm_ but not sure how exactly. Is it even possible?

Could someone give me a hint on how to achieve this?

**Comment by arkodg** [comment]
you could get this values from your access logs https://gateway.envoyproxy.io/docs/tasks/observability/proxy-accesslog/

**Reply by skarasev** [reply]
Thats a shame. The idea of parsing  a ton of logs to aggregate such data does not look good.
Is there a no way to modify the metrics output? I am coming from Istio Telemetry addon, but maybe there is some Envoy-native way to do it?
